### ISSUE TITLE
New name for MacEFT

### ISSUE THREAD

**Issue by pubkraal** [pubkraal/Eden#1]
Hi. We need a new name.

The idea of this project started out as a Mac version of the Eve Fitting Tool for Windows. We've grown beyond that. Mostly because of sheer enthusiasm and also a little because Mac Eve Tools is now defunct, or so I read. Basically this means that MacEFT now also contains mechanics for displaying advanced character information (learned skills, skill queue, market orders etc) and a placeholder (at the moment of writing this post) for skill plans.

In short, the name "MacEFT" doesn't cover the load of awesome the application will deliver to Eve players on Macs around the world. We need a new one.

Suggestions known right now are:
- Snake
- Adam

Obviously they're both references to Eve being unable to resist the Apple that fell from the tree. Eve, Apple, get it? :v. I'm welcoming more ideas and arguments against these names so we can make a decision that befits the application.

**Comment by pubkraal** [comment]
Also read "Eden" on reddit.com. I like it.

**Comment by themayarose** [comment]
Can we settle on Eden then? :) That was my favourite suggestion too.

**Comment by pubkraal** [comment]
sure,.

**Comment by pubkraal** [comment]
The repository will be updated on friday. Please set all repositories to geonetix/Eden instead of geonetix/MacEFT! Changing references to MacEFT in the source code can commence at any time.
